kernel/syscall/common.rs

```diff
@@ -5,13 +5,13 @@ pub const SYS_BRK: usize = 45;
 pub const SYS_CHDIR: usize = 12;
 pub const SYS_CLOSE: usize = 6;
 pub const SYS_CLONE: usize = 120;
-    pub const CLONE_VM: usize = 0x100;
-    pub const CLONE_FS: usize = 0x200;
-    pub const CLONE_FILES: usize = 0x400;
-    pub const CLONE_VFORK: usize = 0x4000;
```

Comment:
Just a guess, I think @jackpot51 here explicitly had these constants indented to fit his style.